### DISCUSSION TITLE
Preview Release #2, Gaming Edition

### DISCUSSION THREAD

**Discussion by wheremyfoodat** [wheremyfoodat/Panda3DS#185] [Announcements]
- Much better compatibility due to lots of emulation fixes
- Graphics fixes (Proper-ish screens!)
- Performance fixes
- Better user experience
- Optional Discord RPC support
- MacOS app bundles!
- Sort of working Linux builds this time
- more

Shout out to everyone who contributed to this release.

<hr /><em>This discussion was created from the release <a href='https://github.com/wheremyfoodat/Panda3DS/releases/tag/v0.2-beta'>Preview Release #2, Gaming Edition</a>.</em>

**Comment by liuk7071** [comment]
so good

**Reply by ghost** [reply]
so good

**Comment by wheremyfoodat** [comment]
So good

**Reply by ghost** [reply]
so good

**Comment by KostaSaizo7** [comment]
![sogood-jamesfranco](https://github.com/wheremyfoodat/Panda3DS/assets/39852873/8a432ffe-0ec7-4551-8984-c0a15afd0129)

**Comment by OFFTKP** [comment]
So good

**Reply by wheremyfoodat** [reply]
rhappy

**Reply by liuk7071** [reply]
rhappy

**Comment by wheremyfoodat** [comment]
Updated the release to make positional lights not pink

**Comment by Wunkolo** [comment]
pamda

**Comment by wheremyfoodat** [comment]
pamda!

**Comment by LaReunion974** [comment]
nice 👍

**Comment by tgsm** [comment]
so good

**Comment by patataofcourse** [comment]
panda 2 electric boogaloo

**Comment by ameliajhons** [comment]
Great work i inspired can you please assist me with playing [fifa apk](https://fifaapk.com/) i am a fifa addict

**Comment by LuisPerss** [comment]
¿Estos son los avances de pc o los de android señores??

**Comment by LuisPerss** [comment]
I know gentleman, it was just a question for the team
